<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClockExample</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widgetTextClock" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>300</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>2</number>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 0, 255);</string>
    </property>
    <widget class="QLabel" name="labelHora">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>311</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>55</pointsize>
      </font>
     </property>
     <property name="toolTipDuration">
      <number>1</number>
     </property>
     <property name="text">
      <string>00:00:00</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelData">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>130</y>
       <width>331</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>55</pointsize>
      </font>
     </property>
     <property name="text">
      <string>01/01/01</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>ButtonClicked()</slot>
 </slots>
</ui>
